### DISCUSSION TITLE
MathJax /Katex vdots do not render properly

### DISCUSSION THREAD

**Discussion by jesgp** [requarks/wiki#3530] [Error / Bug Report]
**Describe the bug**
The vdots command displays a strange vertical bar instead of the three dots.

**To Reproduce**
1. Create a new wiki page (with math display render activated)
2. Enter as content:

```
$\vdots$
$$\vdots$$
```

**Expected behavior**
The vertical lines should be displayed instead of the strange block.

**Screenshots**

Code:
![image](https://user-images.githubusercontent.com/658179/84146933-c9593000-aa5c-11ea-8ce0-fd42d72cf46c.png)

Behavior:
![image](https://user-images.githubusercontent.com/658179/84146968-d9710f80-aa5c-11ea-90d4-5b2a1918a441.png)

**Host Info (please complete the following information):**
 - OS: Heroku deployed as WikiJS template (Linux - Alpine Linux (linux) 3.11.6 x64)
 - Wiki.js version: 2.4.105
 - Database engine: postgresql 12.3

**Comment by NGPixel** [comment]
Please post an example as text, so that it can be reproduced.

**Comment by jesgp** [comment]
1. Create a new Markdown page
2. Enter as text the following:

```
$\vdots$
$$\vdots$$
```

3. The rendering will show up as the image I've pasted.

Please let me know if you need more directions to reproduce.

Thanks for taking a look!

**Reply by Penguinlay** [reply]
The same for me. I use Mathjax and the above bug is in editor preview.

LaTeX Code
```markdown
$$
B =
\begin{bmatrix}
    a_{11} \\
    a_{21} \\
    \vdots \\
    a_{n1}
\end{bmatrix}
$$
```


Editor View
<img width="103" alt="CleanShot 2022-07-04 at 20 50 58@2x" src="https://user-images.githubusercontent.com/13214144/177246052-5daf9c6d-0e98-4959-9361-8e70621dcb18.png">



In end user view, nothing shows up for any of LaTeX SVG.
<img width="755" alt="CleanShot 2022-07-04 at 20 53 55@2x" src="https://user-images.githubusercontent.com/13214144/177246332-3dc45c1d-b2fd-4046-b63a-67930f75feba.png">

Wiki.js Version
"version": "2.5.285"

**Reply by Penguinlay** [reply]
Basically with mathjax, every svg render of latex are transparent block like the above.

**Reply by Penguinlay** [reply]
I can confirm the same issue with KaTeX.

**Comment by sam-artuso** [comment]
I'm experiencing the same issue!

**Comment by howsmyanimeprofilepicture** [comment]
![image](https://github.com/requarks/wiki/assets/75282888/864bf1a1-9463-4a8c-be85-cca3dc1d46d9)

To solve this issue, I add the following css into the CSS override option.

```css
.mord.rule {
    display: none;
}
```

**Reply by figonzal1** [reply]
Works! Thanks

**Comment by jesgp** [comment]
In the meantime, I moved away from Wiki.js. Is the math rendering in general terms better now? I experience some other issues.
